src/building/bootstrapping.md

```diff
@@ -448,3 +463,66 @@ This is an incomplete reference for the outputs generated by bootstrap:
 | copy `rustdoc`                                         | `build/HOST/stage2/bin`                                         |
 
 `--stage=2` stops here.
+
+### Clarification of build command's stdout
+
+In this part, we will investigate the build command's stdout in an action
+(similar, but more detailed and complete documentation compare to topic above).
+When you execute `x.py build` command, the build output will be something
+like the following:
+
+```text
+Building stage0 std artifacts
+Copying stage0 std
+Building stage0 test artifacts
+Copying stage0 test
+Building stage0 compiler artifacts
+Copying stage0 rustc
+Assembling stage1 compiler
+Building stage1 std artifacts
+Copying stage1 std
+Building stage1 test artifacts
+Copying stage1 test
+Building stage1 compiler artifacts
+Copying stage1 rustc
+Assembling stage2 compiler
+Uplifting stage1 std
+Uplifting stage1 test
+Uplifting stage1 rustc
+```
+
+#### Building stage0 {std,test,compiler} artifacts
+
+These steps use the provided (downloaded, usually) compiler to compile the
+local Rust source into libraries we can use.
+
+#### Copying stage0 {std,test,rustc}
+
+This copies the build output from Cargo into
+`build/$HOST/stage0-sysroot/lib/rustlib/$ARCH/lib`.
+[comment]: FIXME: this step's documentation should be expanded -- the information already here may be incorrect.
+
+#### Assembling stage1 compiler
+
+This copies the libraries we built in "building stage0 ... artifacts" into
+the stage1 compiler's lib directory. These are the host libraries that the
+compiler itself uses to run. These aren't actually used by artifacts the new
+compiler generates. This step also copies the rustc and rustdoc binaries we
+generated into `build/$HOST/stage/bin`.
+
+The stage1/bin/rustc is a fully functional compiler, but it doesn't yet have
+any libraries to link built binaries or libraries to. The next 3 steps will
+provide those libraries for it; they are mostly equivalent to constructing
+the stage1/bin compiler so we don't go through them individually.
+
+#### Uplifting stage1 {std,test,rustc}
```

Comment:
Hmm, all this info looks useful, but it seems redundant with the "Stages of bootstrapping" section at the top. I'm not sure what to do here.

Reply:
> Hmm, all this info looks useful,

Yes, it explains a lot of the build process.

> but it seems redundant with the "Stages of bootstrapping" section at the top. I'm not sure what to do here.

It's a little bit(maybe more than little bit) detailed version of it. Do you think this should be under the `bootstrap/README.md` ?